apps/backend/src/libs/modules/server-application/base-server-application.ts

```diff
@@ -214,6 +214,7 @@ class BaseServerApplication implements ServerApplication {
 					token,
 					userService,
 					whiteRoutes: WHITE_ROUTES,
+					whiteRoutesPrefix: `/${api.version}`,
```

Comment:
Add prefix directly in white routes constant

Reply:
Something like this?
```ts
await this.app.register(authorizationPlugin, {
  token,
  userService,
  whiteRoutes: WHITE_ROUTES.map((whiteRoute) => `/${api.version}${whiteRoute}`),
});

```

Or directly on the constant value?

Reply:
Is there a constant for api version?
```ts
import { AuthApiPath } from "~/modules/auth/auth.js";

import { APIPath } from "../enums/enums.js";

const WHITE_ROUTES: string[] = [
	APIPath.AUTH,
	`${APIPath.AUTH}${AuthApiPath.SIGN_IN}`,
	`${APIPath.AUTH}${AuthApiPath.SIGN_UP}`,
].map((whiteRoute) => `/v1${whiteRoute}`);

export { WHITE_ROUTES };

```

Reply:
No, there's no constant. You can create a helper function, which will check if a route is a white route. For that you can use Regex - something like /api/v{number}/endpoint